### ISSUE TITLE
Add support for "config paths" to the rc API

### ISSUE THREAD

**Issue by Jwink3101** [rclone/rclone#7568]
#### The associated forum post URL from `https://forum.rclone.org`

https://forum.rclone.org/t/core-command-does-not-respect-flags-from-rcd/43970/2

#### What is your current rclone version (output from `rclone version`)?

```
rclone v1.65.1
- os/version: darwin 14.2.1 (64 bit)
- os/kernel: 23.2.0 (arm64)
- os/type: darwin
- os/arch: arm64 (ARMv8 compatible)
- go/version: go1.21.5
- go/linking: dynamic
- go/tags: cmount
```

#### What problem are you are trying to solve?

Ascertain (*not* set) the config path, the rclone cache dir, and the rclone temp dir

#### How do you think rclone should be changed to solve that?

Add an endpoint `config/paths` to the RC.

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by ncw** [comment]
Please give this a go

[v1.66.0-beta.7622.b0e9d8662.fix-7568-config-paths](https://beta.rclone.org/branch/fix-7568-config-paths/v1.66.0-beta.7622.b0e9d8662.fix-7568-config-paths/) on branch [fix-7568-config-paths](https://github.com/rclone/rclone/tree/fix-7568-config-paths) (uploaded in 15-30 mins)

### config/paths: Reads the config file path and other important paths. {#config-paths}

Returns a JSON object with the following keys:

- config: path to config file
- cache: path to root of cache directory
- temp: path to root of temporary directory

Eg

    {
        "cache": "/home/USER/.cache/rclone",
        "config": "/home/USER/.rclone.conf",
        "temp": "/tmp"
    }

See the [config paths](/commands/rclone_config_paths/) command for more information on the above.

**Authentication is required for this call.**

**Comment by Jwink3101** [comment]
This works for me! I tested it with both flag and environment specification of paths and it correctly represents them!

**Comment by ncw** [comment]
Thanks for testing :-)

I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org/) in 15-30 minutes and released in v1.66
